homeassistant/components/cover/myq.py

```diff
@@ -89,6 +91,16 @@ def is_closed(self):
         """Return true if cover is closed, else False."""
         return self._status == STATE_CLOSED
 
+    @property
+    def is_closing(self):
+        """Return if the cover is closing or not."""
+        return self._status == STATE_CLOSING
```

Comment:
We should add a translation dict between the myq status and home assistant state constants. This will make sure we control the coupling, and avoid breakage if a constant changes.

Reply:
I think I follow, something like this then?
```python
MYQ_TO_HASS = {
    'closed': STATE_CLOSED,
    'closing': STATE_CLOSING,
    'opening': STATE_OPENING
}

    @property
    def is_closing(self):
        """Return if the cover is closing or not."""
        return MYQ_TO_HASS[self._status] == STATE_CLOSING
```

Reply:
Exactly! :+1: